graviti/dataframe/frame.py

```diff
@@ -154,9 +154,8 @@ def __setitem__(self, key: str, value: Union[Iterable[Any], _C]) -> None:
         self._columns[key] = column
         if key not in self._column_names:
             self._column_names.append(key)
-        # TODO: Need add corresponding operations.
         if self.operations is not None:
-            self.operations.append(UpdateData(self[[key]]))
+            self.operations.extend((UpdateSchema(self.schema), UpdateData(self[[key]])))
```

Comment:
will append twice faster?

Reply:
extend tuple will be a little faster than append twice